android-components/components/feature/accounts/src/main/java/mozilla/components/feature/accounts/FxaWebChannelFeature.kt

```diff
@@ -363,11 +370,33 @@ class FxaWebChannelFeature(
             return null
         }
 
+        /**
+         * Handles the [COMMAND_DELETE] event from the web-channel.
+         */
+        private fun processDeleteCommand(accountManager: FxaAccountManager, payload: JSONObject): JSONObject? {
+            return try {
+                val data = payload.getJSONObject("data")
+                val uid = data.getString("uid")
+                if (uid.isNotEmpty() && uid == accountManager.accountProfile()?.uid) {
+                    CoroutineScope(Dispatchers.Main).launch {
+                        accountManager.onAccountDeleted()
+                    }
+                }
+
+                null
+            } catch (e: JSONException) {
+                // TODO ideally, this should log to Sentry.
```

Comment:
Let's leave a TODO with link to a filed bug if we're not going to do it in this patch, other it never gets done. :)